piety-promoted-v3/modernized/13-chapter-13.adoc

```diff
@@ -302,12 +301,11 @@ This is not often the case; but I desire to be content and thankful.`'
 
 She departed the 31st of the Fifth month, 1804.
 
-A short sketch of the engagement of mind of Elizabeth Millis,
-referred to in the foregoing account.
+=== A short sketch of the engagement of mind of Elizabeth Millis, referred to in the foregoing account.
```

Comment:
Not sure if this should be a sub heading or not.

Reply:
sort of a tough call, but I think you got it right. 👍